### PR TITLE
how to create a sudo user in ubuntu

### PR DESCRIPTION
**Pull request by MatthiasLiszt**
first commit of mine ... sorry in case I made any mistakes

### CODE REVIEW

File: src/pages/linux/how-to-create-a-sudo-user-on-ubuntu/index.md [hunk 1]
```diff
@@ -3,13 +3,23 @@ title: How to Create a Sudo User on Ubuntu
...
+Make sure that &lt;username&gt; is actually an existing user. In case it isn't just type in 
+  <pre>adduser &lt;username&gt;</pre> 
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
 
 <!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
 
 #### More Information:
-<!-- Please add any articles you think might be helpful to read before writing the article -->
+I recommend to check out articles and question on <a href="askubuntu.com">askubuntu.com</a> .
```
Comment (Bouncey) [inline src/pages/linux/how-to-create-a-sudo-user-on-ubuntu/index.md]: `<a href='...' target='_blank' rel='nofollow'>`

File: src/pages/linux/how-to-create-a-sudo-user-on-ubuntu/index.md [hunk 2]
```diff
@@ -3,13 +3,23 @@ title: How to Create a Sudo User on Ubuntu
...
+  <pre>adduser &lt;username&gt;</pre> 
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
 
 <!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
 
 #### More Information:
-<!-- Please add any articles you think might be helpful to read before writing the article -->
+I recommend to check out articles and question on <a href="askubuntu.com">askubuntu.com</a> . 
+Especially <a href="https://askubuntu.com/questions/7477/how-can-i-add-a-new-user-as-sudoer-using-the-command-line">this question</a>.
```
Comment (Bouncey) [inline src/pages/linux/how-to-create-a-sudo-user-on-ubuntu/index.md]: `<a href='...' target='_blank' rel='nofollow'>`

File: src/pages/linux/how-to-create-a-sudo-user-on-ubuntu/index.md [hunk 3]
```diff
@@ -3,13 +3,23 @@ title: How to Create a Sudo User on Ubuntu
...
 ## How to Create a Sudo User on Ubuntu
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/linux/how-to-create-a-sudo-user-on-ubuntu/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+Your installed Linux distribution might have already a root account. This is how you can let an exisiting
+user use the <i>sudo</i> command. The sudo command is required if you want to install software or better 
+packages like for instance the npm package. 
+
+In order to do this you must login as root or being able to sudo. 
+Afterwards you write 
+<pre>usermod -a -G sudo <username></pre>
```
Comment (Bouncey) [inline src/pages/linux/how-to-create-a-sudo-user-on-ubuntu/index.md]: Can you escape these angle brackets as well please

File: src/pages/linux/how-to-create-a-sudo-user-on-ubuntu/index.md [hunk 4]
```diff
@@ -3,13 +3,23 @@ title: How to Create a Sudo User on Ubuntu
 ---
 ## How to Create a Sudo User on Ubuntu
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/linux/how-to-create-a-sudo-user-on-ubuntu/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+Your installed Linux distribution might have already a root account. This is how you can let an exisiting
+user use the <i>sudo</i> command. The sudo command is required if you want to install software or better
```
Comment (Bouncey) [inline src/pages/linux/how-to-create-a-sudo-user-on-ubuntu/index.md]: `<em>` tags would be better instead of `<i>`

**Comment by systimotic** [comment]
I apologize, I just merged https://github.com/freeCodeCamp/guides/pull/340 without checking if there was another PR on the same thing first. I'm closing this, but I'd like to ask @MatthiasLiszt to take a look at the current article to see what improvements you could make to it. It seems you are quite knowledgeable on this, I think you could add a lot of good information to the new article.

**Pull request closed without merging**